src/main/java/pro/sky/java/course2/examinerservice/domain/Question.java

```diff
@@ -0,0 +1,42 @@
+package pro.sky.java.course2.examinerservice.domain;
+
+import java.util.Objects;
+
+public class Question {
+    private String question;
+    private String answer;
```

Comment:
```suggestion
    private final String question;
    private final String answer;
```
поскольку данные поля не предназначены для изменения после создания объекта